src/Sarif.Viewer.VisualStudio/Views/StringResources.xaml

```diff
@@ -0,0 +1,62 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:system="clr-namespace:System;assembly=mscorlib">
+
+    <!-- SARIF Viewer Control -->
```

Comment:
What do you think of the idea of pushing the string resources for each category down into the corresponding resource file, rather then mixing them all into a single StringResources.xaml file? Like, the Stacks-related strings go into StacksStyles.xaml? Alternatively, separate them into separate files like StacksStrings.xaml, so that Stacks.xaml would include both StacksStyles and StacksStrings?

Reply:
The idea is that the strings file can be sent to a localization service, and they return the same file for each language. Isolating the strings makes this process clean and simple.

Reply:
@EasyRhinoMSFT  Ok. I leave it to you to make the right trade-off here between separation of concerns, on the one hand, and ease of localization, on the other. _If_ it were just as easy for the localizers to handle a zip file containing several separate resource files as it is for them to deal with a single resource file, I'd like my proposal better. But you know the process and I don't, so do whatever it right.

Reply:
Split the difference, created separate files for each.